pkg/flare/archive_docker.go

```diff
@@ -56,3 +60,35 @@ func zipDockerSelfInspect(tempDir, hostname string) error {
 	_, err = w.Write(serialized)
 	return err
 }
+
+func zipDockerPs(tempDir, hostname string) error {
+	du, err := docker.GetDockerUtil()
+	if err != nil {
+		return err
+	}
+	options := types.ContainerListOptions{All: true}
+	containerList, err := du.RawContainerList(options)
+	if err != nil {
+		return err
+	}
+
+	// Opening out file
+	f := filepath.Join(tempDir, hostname, "docker_ps.log")
+	file, err := os.Create(f)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	w := tabwriter.NewWriter(file, 20, 0, 3, ' ', 0)
+
+	fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tSTATUS\tPORTS\tNAMES\t")
+	// Removed CREATED as it only shows a timestamp in the API
+	for _, c := range containerList {
+		fmt.Fprintf(w, "%s\t%s\t%q\t%s\t%v\t%v\t\n",
+			c.ID[:12], c.Image, c.Command, c.Status, c.Ports, c.Names)
+		w.Flush()
+	}
+
+	return err
```

Comment:
`return nil` at the end is usually more explicit

Reply:
👍